src/core/ApolloClient.ts

```diff
@@ -511,10 +512,38 @@ export class ApolloClient implements DataProxy {
   >(
     options: MutationOptions<TData, TVariables, TCache>
   ): Promise<MutateResult<MaybeMasked<TData>>> {
-    if (this.defaultOptions.mutate) {
-      options = mergeOptions(this.defaultOptions.mutate, options);
+    const optionsWithDefaults = mergeOptions(
+      compact(
+        {
+          fetchPolicy: "network-only" as MutationFetchPolicy,
+          errorPolicy: "none" as ErrorPolicy,
+        },
+        this.defaultOptions.mutate
+      ),
+      options
+    ) as MutationOptions<TData, TVariables, TCache> & {
+      fetchPolicy: MutationFetchPolicy;
+      errorPolicy: ErrorPolicy;
+    };
+
+    if (__DEV__) {
+      invariant(
+        optionsWithDefaults.mutation,
+        "mutation option is required. You must specify your GraphQL document in the mutation option."
```

Comment:
```suggestion
        "The `mutation` option is required. Please provide a GraphQL document in the `mutation` option."
```

I tend to like using backticks for code-related references. We also tend to use "please" in other message so I figure we could format this similarly.